src/sentry/api/serializers/models/organization.py

```diff
@@ -53,6 +56,37 @@ def serialize(self, obj, attrs, user):
 
         status = OrganizationStatus(obj.status)
 
+        # Retrieve all registered organization features
+        org_features = features.all(feature_type=OrganizationFeature).keys()
+        feature_list = set()
+
+        for feature_name in org_features:
+            if not feature_name.startswith('organizations:'):
+                continue
+            if features.has(feature_name, obj, actor=user):
+                # Remove the organization scope prefix
+                feature_list.add(feature_name[len('organizations:'):])
```

Comment:
You call `len()` multiple times on a fixed string. You could hoist that above the loop to save some time. :golf:

Reply:
On second thoughts, will prefer to not touch this for now

Reply:
Yeah this was my doing 🤷‍♂️